__tests__/app.test.js

```diff
@@ -26,41 +22,47 @@ describe('/api/topics', () => {
             })
         })
     })
-    test('returns 404 if no topics found', ()=> {
-        return db.query(` DELETE FROM comments; DELETE FROM articles; DELETE FROM topics`).then(()=> {
-            return request(app)
-            .get('/api/topics')
-            .expect(404)
-            .then(({body}) => {
-                expect(body).toEqual({msg: "No Topics Found"})
-            })
-        })
-    })
-    test('returns 404 if table does not exist', ()=> {
-        return db.query(` DROP TABLE comments; DROP TABLE articles; DROP TABLE topics`).then(()=> {
-            return request(app)
-            .get('/api/topics')
-            .expect(404)
-            .then(({body}) => {
-                expect(body).toEqual({msg: "Table Does Not Exist"})
-            })
-        })
-    })
 });
 
 describe('/api', ()=> {
-    test('returns status code 200', () => {
-        return request(app)
-        .get('/api')
-        .expect(200)
-    });
     test('returns a JSON object matching the contents of endpoints.json', () => {
         return fs.readFile('endpoints.json','utf-8').then((endpoints) => {
             return request(app)
             .get('/api')
+            .expect(200)
             .then(({body})=> {
                 expect(body.endpoints).toEqual(endpoints)
             })
         })
     });
-})
\ No newline at end of file
+})
+
+describe('/api/articles/:article_id', () => {
+    test('Returns an article with the correct id', () => {
+        const article_id = 1
+        return request(app)
+        .get(`/api/articles/${article_id}`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article.article_id).toEqual(article_id)
+        })
+    });
```

Comment:
It's good that you're testing for the exact article id to come back, but what about the other properties?

To test the properties of the response object, you might find it helpful to use a flexible matcher like [.toMatchObject](https://jestjs.io/docs/expect#expectobjectcontainingobject) or [objectContaining](https://jestjs.io/docs/expect#expectobjectcontainingobject) combined with `expect.any(*data type*)` have a look into this! (also, a benefit of .toMatchObject is that it ignores extra properties which leaves room for expansion that won't break the test later on)